packages/node-cli/src/commands/project/deploy/DeployAction.js

```diff
@@ -70,17 +75,51 @@ module.exports = class DeployAction extends BaseAction {
 				flags.push(COMMAND.FLAGS.APPLY_CONTENT_PROTECTION);
 			}
 
-			const sdkParams = CommandUtils.extractCommandOptions(params, this._commandMetadata);
+			if (params[COMMAND.FLAGS.PREVIEW]) {
```

Comment:
Since the execute command can now either call preview or deploy, maybe it would be a good idea to move the preview and deploy parts of the code to their own functions.